<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>泪眼问花花不语，乱红飞过秋千去。</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#Dialog {

	
	border-image: url(:/Sources/778242_1.png);
} </string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>231</width>
     <height>285</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="Mtest1LE">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">  
font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；</string>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="ScoreLB2">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>第二次成绩：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="Mtest2LE">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">  
font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="ScoreLB3">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>第三次成绩：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="Mtest3LE">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">  
font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="ScoreLB4">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>第四次成绩：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="Mtest4LE">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">  
font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="ScoreLB5">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>第五次成绩：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="Mtest5LE">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">  
font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="ScoreLB6">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>平时成绩：</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="Mtest6LE">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">  
font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ScoreLB1">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>第一次成绩：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Time">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>Hi~ 这是您大学生活的第%dd，天！祝您心情愉快</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STIX</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作业完成情况</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>301</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>   font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；
  </string>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="homework">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>缺交次数：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="HWtotal">
      <property name="styleSheet">
       <string notr="true">   font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；
  </string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="HomeworkLB1">
      <property name="styleSheet">
       <string notr="true">   font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；
  </string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLineEdit" name="HomeworkLB2">
      <property name="styleSheet">
       <string notr="true">   font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；
  </string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="HomeworkLB3">
      <property name="styleSheet">
       <string notr="true">   font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；
  </string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLineEdit" name="HomeworkLB4">
      <property name="styleSheet">
       <string notr="true">   font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；
  </string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLineEdit" name="HomeworkLB5">
      <property name="styleSheet">
       <string notr="true">   font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；
  </string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLineEdit" name="HomeworkLB6">
      <property name="styleSheet">
       <string notr="true">   font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；
  </string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLineEdit" name="HomeworkLB7">
      <property name="styleSheet">
       <string notr="true">   font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；
  </string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLineEdit" name="HomeworkLB8">
      <property name="styleSheet">
       <string notr="true">   font: 75 10pt &quot;微软雅黑&quot;;
 background-color: rgba(255, 255, 255, 0%);

   border:1px solid #b9babb;
   border-width:0;border-style:outset；
  </string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="SubChoose">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>请选择科目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>英语</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电工</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>物理</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>离散数学</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>同学</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>196</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_5">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>GPA查询</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>去广告翻译</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="EnglishPB">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>英语</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MathPB">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>高数</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
